CONTRIBUTING.md

```diff
@@ -0,0 +1,107 @@
+
+
+4. In your text editor, Select the file related to the issue you want to fix, and make your changes.
+   
+   ⚠️**Note:** To contribute to a documentation, you would need to have Markdown knowledge. Visit [here](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) to read about GitHub Markdown and [here](https://www.markdowntutorial.com/) to practice.
+   
+5. Commit your changes with a descriptive commit message [using conventional commits](https://www.conventionalcommits.org/en/v1.0.0/).
+
+
+    #### We highly encourage the use of conventional commits. Here are some examples:
+
+     - **feat:** Use this when introducing a new feature.
+     - **fix:**  Use this when resolving any issues in the codebase.
+     - **chore:** Use this when adding new links/resources or making minor changes.
+    (e.g, `chore: Add database credentials`)
+     - Please ensure that your **commit messages are concise and clear**.
+     - Write commit messages in the **present tense**, as they reflect the current state of the codebase after the changes have been applied.  
+    <br/>
+6. Push your branch to GitHub
+   
+    ```
+    git push origin <my-branch>
+    ```
+7. [Open a Pull Request](https://help.github.com/articles/using-pull-requests/) with a clear title and description matching the issue you fixed.
+
+
+    ⚠️**Note:** To ensure a smoother review process, kindly  use the following format for the Pull Request title:`[chore]`, `[feat]`, or `[fix]`, followed by a descriptive title. 
+    
+    For instace, a feat-related change could have a title like `[feat] setup pasword reset`. This helps us categorize and understand the nature of the changes made in each PR.
+
+   - Link the issue you have resolved in the Pull Request Template using the following syntax:
+   - If your Pull Request fixes issue #25, add `Fixes #25` to the description.
+   - If your Pull Request addresses multiple issues, list them using the same syntax (`Fixes #23, Fixes #15`) however, we'll advise you create seperate PRs for each.  
+  <br/>
+    This helps us track and automatically close the relevant issue when your Pull Request is merged.
+
+## Contribution Etiquettes
+
+- When you pick an issue to fix, we recommended that you review the comment thread first to check if someone is already working on it. In case no one has claimed it yet, kindly leave a comment stating your intention to work on it. This way, others will be aware, and we can avoid accidental duplication of effort.
+
+- If someone has shown interest in fixing an issue but doesn't follow up for a specific period, let's say 2-3 weeks, it's permissible to pick up the same issue later. However, please ensure that you leave a comment to inform others of your intention before proceeding.
+
+
+   #### Thank you for reading the guide. We look forward to merging your contributions.
```

Comment:
We may need to check the indentation here.

Reply:
Done